docs/subnet.md

```diff
@@ -57,22 +57,17 @@ This section provides instructions for spawning a simple subnet with a single va
 To run a subnet the first thing is to configure and create the subnet actor that will govern the subnet's operation.
 
 ```bash
-$ ./bin/ipc-agent subnet create -p <parent-id> -n <subnet-name> --min-validator-stake 1 --min-validators <num-validators> --finality-threshold <number-epochs> --check-period <epochs-between-checks>
+$ ./bin/ipc-agent subnet create --parent <parent> --name <name> --min-validator-stake <min_validator_stake> --min-validators <min-validators> --bottomup-check-period <bottomup-check-period> --topdown-check-period <topdown-check-period>
```

Comment:
These commands were broken due to wrong arguments. I fixed them in this repo but was wondering whether they might be found elsewhere. I don't think so, but just a note.